src/test/java/org/folio/rest/impl/OaiPmhImplTest.java

```diff
@@ -684,28 +684,28 @@ void getOaiListVerbResumptionFlowStarted(MetadataPrefix metadataPrefix, VerbType
 
     OAIPMH oaipmh = verify200WithXml(request, verb);
 
-    verifyListResponse(oaipmh, verb, 10);
+    verifyListResponse(oaipmh, verb, 11);
 
     ResumptionTokenType resumptionToken = getResumptionToken(oaipmh, verb);
 
     assertThat(resumptionToken, is(notNullValue()));
-    assertThat(resumptionToken.getCompleteListSize(), is(equalTo(BigInteger.valueOf(100))));
+    assertThat(resumptionToken.getCompleteListSize(), is(equalTo(BigInteger.valueOf(64))));
     assertThat(resumptionToken.getCursor(), is(equalTo(BigInteger.ZERO)));
     assertThat(resumptionToken.getExpirationDate(), is(notNullValue()));
 
     String resumptionTokenValue =
       new String(Base64.getUrlDecoder().decode(resumptionToken.getValue()), StandardCharsets.UTF_8);
     List<NameValuePair> params = URLEncodedUtils.parse(resumptionTokenValue, StandardCharsets.UTF_8);
-    assertThat(params, is(hasSize(14)));
+    assertThat(params, is(hasSize(13)));
 
     assertThat(getParamValue(params, METADATA_PREFIX_PARAM), is(equalTo(metadataPrefix.getName())));
     assertThat(getParamValue(params, FROM_PARAM), is(equalTo(PARTITIONABLE_RECORDS_DATE_TIME)));
     assertThat(getParamValue(params, UNTIL_PARAM), is((notNullValue())));
     assertThat(getParamValue(params, SET_PARAM), is(equalTo("all")));
-    assertThat(getParamValue(params, OFFSET_PARAM), is(equalTo("10")));
-    assertThat(getParamValue(params, TOTAL_RECORDS_PARAM), is(equalTo("100")));
+    assertThat(getParamValue(params, OFFSET_PARAM), is(equalTo("11")));
```

Comment:
The pure values, used as expected results should be set up as constants, which describe meaning.
Such an approach increases the code understanding during the research/investigation stages and decreases efforts for support.